components/date-picker/date-input.js

```diff
@@ -14,6 +14,7 @@ export default class DateInput extends Component {
     text: PropTypes.string,
     hoverDate: dateType,
     date: dateType,
+    time: PropTypes.string,
```

Comment:
As far as I know, "date" prop is a JS/MomentJS Date, so it could hold both Date and Time inside. Maybe we don't have to split this single entity to "date" and "time"?

Reply:
I'll do the math about this one

Reply:
The person I'm doing this component for would like to see date and time as different entities. 
Also i think
- having the ability to chose whether to specify time and date or date only gives flexibility and keeps things backwards compatible
- since with range=true usecase time is unnecessary and is not supposed to work, merging date and time would create inconsistency ({date} would in fact be date and time and {from} and {to} are just dates)

Reply:
Okay, makes sense. Thank you for investigating this!